internal/venice-common/src/main/resources/avro/StoreMetaValueWriteOpRecord.avsc

```diff
@@ -150,6 +150,40 @@
               "doc": "Properties related to Hybrid Store behavior. If absent (null), then the store is not hybrid.",
               "default": null
             },
+            {
+              "name": "views",
+              "doc": "A map of views which describe and configure a downstream view of a venice store. Keys in this map are for convenience of managing configs.",
+              "type": {
+              "type":"map",
+              "values": {
+                "name": "StoreViewConfig",
+                "type": "record",
+                "doc": "A configuration for a particular view.  This config should inform Venice leaders how to transform and transmit data to destination views.",
```

Comment:
As this config is for describe and configure a downstream view, can we say we have 2 views after this change: 1. Normal 2. ChangeCapture. If so, how about we have Normal as default. 
Besides, just wondering what kinds of other possible viewTypes we might have in future?

Reply:
What would a normal view be?

Reply:
So really it'd be any kind of stored procedure sort of thing.  It could be a projection, it could be an index inversion, it could be building up and backing up rocksdb files, it could be counting occurrences.

https://www.w3resource.com/mysql/mysql-procedure.php

Reply:
I mean normal here is for our current no downstream view as default. Just curious we might have multiple co-existing views for a specific store?

Reply:
Yes, I'm anticipating there to be an arbitrary number of views per store depending on what features are getting leveraged.